build/webpack.config.umd.ts

```diff
@@ -1,15 +1,15 @@
 import config from '../config'
 import webpackConfig from '../webpack.config'
 
-const pkg = require('../package.json')
+const pkg = require('../packages/react/package.json')
```

Comment:
seems that intent was to require `pkg` by `packageName` as well (instead of `react` being hardcoded)